languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -134267,6 +134415,16 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
         </rule>
     </category>
     <category id="CAT_TYPOGRAPHIE" name="Typographie" type="typographical">
+        <rule id="DONT_ACTE" name="donc-dont acte" default="temp_off">
```

Comment:
Not sure to get why this rule is in the TYPOGRAPHIE category, it's about changing _donc acte_ into _dont acte_, closer to the CAT_TOURS_CRITIQUES?

Reply:
My thought behind this was that this error is probably usually a typo: accidentally pressing the c key instead of the t key.
But if you think that French people usually write "donc" instead of "dont" on purpose, then yes, I can move it to CAT_TOURS_CRITIQUES.
Please advise.

Reply:
CAT_TOURS_CRITIQUES it is.